exonum/src/events/network.rs

```diff
@@ -96,6 +116,7 @@ impl Default for NetworkConfiguration {
             tcp_nodelay: true,
             tcp_connect_retry_timeout: 15_000,
```

Comment:
It may make sense to use uniform terminology ("interval" vs "timeout" and "max retries" vs "retry attempts").